resources/cert-manager/README.md

```diff
@@ -0,0 +1,61 @@
+# cert-manager
+
+## Overview
+
+`cert-manager` is a version of [Jetstack's cert-manager](https://cert-manager.io) customized to Kyma's needs.
+
+Quoting from the [official README](https://github.com/jetstack/cert-manager):
+
+>cert-manager is a Kubernetes addon to automate the management and issuance of
+>TLS certificates from various issuing sources.
+>
+>It will ensure certificates are valid and up to date periodically, and attempt
+>to renew certificates at an appropriate time before expiry.
+
+
+For more information check [links](#links) section.
+
+## Prerequisites
+
+- Kubernetes 1.11+
+
+## Details
```

Comment:
Please revert it to the way it was - remove the `## Details` section and change its subsections back to sections.